mlflow/R/mlflow/R/python.R

```diff
@@ -1,13 +1,34 @@
+# Gets path to Python provided as an argument when running the script
+get_bin_from_args <- function(bin) {
+  in_args <- grepl(paste0("--", bin), commandArgs())
+  if (any(in_args)) {
+    path <- commandArgs()[in_args]
+    path <- trimws(strsplit(path, "=")[[1]][2])
+  } else {
+    path <- NULL
+  }
+  path
+}
+
 # Computes path to Python executable within conda environment created for the MLflow R package
 #' @importFrom reticulate conda_list
 get_python_bin <- function() {
-  conda <- mlflow_conda_bin()
-  envs <- conda_list(conda = conda)
-  mlflow_env <- envs[envs$name == mlflow_conda_env_name(), ]
-  if (nrow(mlflow_env) == 0) {
-    stop("MLflow not configured, please run install_mlflow().")
+  if ("--no-conda" %in% commandArgs()) {
+    path <- get_bin_from_args("python")
+    if (is.null(path)) {
+      stop(paste("When using --no-conda option you also need to provide path to",
+                 "Python and MLflow using --python=<path> and --mlflow=<path>"))
+    }
+  } else {
+    conda <- mlflow_conda_bin()
+    envs <- conda_list(conda = conda)
+    mlflow_env <- envs[envs$name == mlflow_conda_env_name(), ]
+    if (nrow(mlflow_env) == 0) {
+      stop("MLflow not configured, please run install_mlflow().")
```

Comment:
Update the error message to include the option of setting python binary explicitly.